tests/test_manage.py

```diff
@@ -286,6 +345,133 @@ def test_create_request_with_granting_access(client):
     assert res.status_code == 201, res.text
 
 
+@pytest.mark.parametrize(
+    "data",
+    [
+        {
+            # provide resource_path to get the default reader policies
+            "username": "requestor_user",
+            "resource_path": "/study/123456",
+            "resource_id": "uniqid",
+            "resource_display_name": "My Resource",
+        },
+        {
+            # provide resource_paths without role_ids to get the default reader policies
+            "username": "requestor_user",
+            "resource_paths": ["/study/123456", "/mds_gateway", "/cedar"],
+            "resource_id": "uniqid",
+            "resource_display_name": "My Resource",
+        },
+        {
+            # resource_paths will take precedence over resource_path
+            "username": "requestor_user",
+            "resource_paths": ["/study/123456", "/mds_gateway", "/cedar"],
+            "resource_path": "/older_study/000111",
+            "resource_id": "uniqid",
+            "resource_display_name": "My Resource",
+        },
+    ],
+)
+def test_create_request_with_resource_path(client, data):
```

Comment:
Same comment here. Maybe this makes more sense in the `test_manage_resource_path.py`